### DISCUSSION TITLE
Reed Solomon decode with c differnet from '1'

### DISCUSSION THREAD

**Discussion by miritom** [mhostetter/galois#214] [General]
Hi @mhostetter,

Thanks for maintaining this library. I started using it recently and it's very useful. 
One thing I noticed is the Reed Solomon decode utility requires c=1 to work properly.
I tried the same example from the documentation, just added "c=2" to the constructor and it failed.
Is this a known limitiation? I saw that there are tests for the "encode" method with different c values, but all "decode" tests use the default, although I dindn't observe any comment on this in the documentation.

**Comment by mhostetter** [comment]
@miritom thank you for raising this issue! I am now tracking this bug in #215.

**Reply by mhostetter** [reply]
@miritom this issue is now fixed! You can install the latest version from `master` like this.

```
$ pip3 uninstall galois
$ pip3 install git+https://github.com/mhostetter/galois.git
```

**Reply by miritom** [reply]
Thanks a lot! I will install and give it a try

**Comment by mhostetter** [comment]
This was fixed in #216.
